components/edit-collective/sections/Members.js

```diff
@@ -231,6 +231,7 @@ class Members extends React.Component {
             <Flex flex="1" justifyContent="space-between" alignItems="center" flexWrap="wrap" mb={2}>
               <Box ml={1} my={1}>
                 <CollectivePickerAsync
+                  inputId="member-collective-picker"
```

Comment:
`renderMember` is called in a loop, so we will end up with multiple input having the same `inputId`

Reply:
I think we can prepend this with the `index` which will resolve this issue. Thanks for pointing it out. 👍🏽